mantle/kola/tests/misc/raid.go

```diff
@@ -114,75 +113,27 @@ func RootOnRaid(c cluster.TestCluster) {
 		c.Fatal(err)
 	}
 
-	checkIfMountpointIsRaid(c, m, "/")
+	util.CheckIfMountpointIsRaid(c, m, "/")
 
 	// reboot it to make sure it comes up again
 	err = m.Reboot()
 	if err != nil {
 		c.Fatalf("could not reboot machine: %v", err)
 	}
 
-	checkIfMountpointIsRaid(c, m, "/")
+	util.CheckIfMountpointIsRaid(c, m, "/")
 }
 
 func DataOnRaid(c cluster.TestCluster) {
 	m := c.Machines()[0]
 
-	checkIfMountpointIsRaid(c, m, "/var/lib/data")
+	util.CheckIfMountpointIsRaid(c, m, "/var/lib/data")
 
 	// reboot it to make sure it comes up again
 	err := m.Reboot()
 	if err != nil {
 		c.Fatalf("could not reboot machine: %v", err)
 	}
 
-	checkIfMountpointIsRaid(c, m, "/var/lib/data")
-}
-
-type lsblkOutput struct {
-	Blockdevices []blockdevice `json:"blockdevices"`
-}
-
-type blockdevice struct {
-	Name       string        `json:"name"`
-	Type       string        `json:"type"`
-	Mountpoint *string       `json:"mountpoint"`
-	Children   []blockdevice `json:"children"`
-}
-
-// checkIfMountpointIsRaid will check if a given machine has a device of type
-// raid1 mounted at the given mountpoint. If it does not, the test is failed.
-func checkIfMountpointIsRaid(c cluster.TestCluster, m platform.Machine, mountpoint string) {
```

Comment:
In general, significant movement of code should be done in a separate commit to make the diff easier to read.

There's no technical requirement to move this code to a separate file.  Since `raid.go` and `boot-mirror.go` are in the same package, they can share private functions.  But I'm okay with the move if you prefer.